app/test_main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from unittest import mock
+import pytest
+from app.main import can_access_google_page
+
+
+@pytest.mark.parametrize("connection_status, valid_url, expected",
+                         [
+                             (True, True, "Accessible"),
```

Comment:
Better to add  `pytest.param` > `id="..."` for better test recognition

Reply:
thanks for advise! did according to comment